lib/wrappers/rust/Cargo.toml

```diff
@@ -6,6 +6,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-contourwall_core = {path = "../../cw-core" , version = "0.1.0" }
+ contourwall_core = {path = "../../cw-core" }
+ 
+#Uncomment the following line if you are developing the wrapper and want the newest version of the core libary
+#contourwall_core = "0.2.1"
+
```

Comment:
this line should be commented: `contourwall_core = {path = "../../cw-core" }`
Not this: `contourwall_core = "0.2.1"`

Reply:
done!